shard.yml

```diff
@@ -1,7 +1,7 @@
 name: ulid
 version: 0.1.3
 
-crystal: "0.36.0, < 2.0.0"
+crystal: ">= 0.36.0, < 2.0.0"
```

Comment:
```suggestion
crystal: "~> 1.0"
```

Reply:
This is to support previous versions. Is also tested in pipeline.

Reply:
https://github.com/place-labs/jwks/pull/4#discussion_r615564659